demo/complex-form.html

```diff
@@ -0,0 +1,95 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../paper-styles/color.html">
+
+<link rel="import"href="../../iron-flex-layout/classes/iron-flex-layout.html">
+<link rel="import" href="../../paper-styles/color.html">
+<link rel="import" href="../../paper-input/paper-input.html">
+<link rel="import" href="../../paper-button/paper-button.html">
+<link rel="import" href="../../gold-cc-input/gold-cc-input.html">
+<link rel="import" href="../../gold-cc-cvc-input/gold-cc-cvc-input.html">
+<link rel="import" href="../../gold-cc-expiration-input/gold-cc-expiration-input.html">
+<link rel="import" href="../../gold-zip-input/gold-zip-input.html">
+
+<dom-module id="complex-form">
+
+  <link rel="import" type="css" href="simple-form.css">
+
+  <style>
+  gold-cc-expiration-input, gold-cc-cvc-input {
+    width: 120px;
+    margin-right: 20px;
+  }
+
+  gold-zip-input {
+    width: 120px;
+  }
+
+  .country {
+    width: 260px;
+    margin-right: 20px;
+  }
+
+  .quantity {
+    width: 120px;
+  }
+
+  h4 {
+    margin-bottom: 0px;
+  }
+
+  paper-button {
+    width: 100%;
+  }
+  </style>
+  <template>
+
+    <div id="form">
```

Comment:
In the future this will be an `iron-form`, not a div, which is why it's in its own labelled div. I can remove it for now if you want. (here and below)